scripts/taco_to_burrito.py

```diff
@@ -151,7 +173,7 @@ def convert_markers(xml_path: str, output_dir: str):
         eprint("Root has {} children instead of the expected 2".format(len(root)))
         return
 
-    if root[0].tag != "MarkerCategory":
+    if root[0].tag != "MarkerCatego6ry":
```

Comment:
This has a 6 in it. In the future, please make sure simple issues like this are corrected before asking for a review.